jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/assemble.sh

```diff
@@ -8,6 +8,5 @@ source "${JBOSS_CONTAINER_WILDFLY_S2I_MODULE}/s2i-core-hooks"
 # inject our overridden maven_s2i_*() functions
 source "${JBOSS_CONTAINER_WILDFLY_S2I_MODULE}/maven-s2i-overrides"
 
-export MAVEN_ARGS_APPEND="${MAVEN_ARGS_APPEND} -Dcom.redhat.xpaas.repo.jbossorg"
```

Comment:
isn't `-Dcom.redhat.xpaas.repo.jbossorg` a magic property that adds Red Hat Maven repository before `mvn package` is invoked?

Reply:
Yes, it has been move to wildfly-s2i: https://github.com/wildfly/wildfly-s2i/pull/226/files#diff-61e961b6eadd77126d79b1c119b6c98dcd2b727bf14263db93ddfe4ee2c7a7d9R3